data/json/monstergroups/eggs.json

```diff
@@ -88,7 +87,12 @@
   {
     "name": "GROUP_EGG_SNAKE",
     "type": "monstergroup",
-    "monsters": [ { "monster": "mon_rattlesnake" } ]
+    "monsters": [ { "monster": "mon_rattlesnake_s" } ]
+  },
+  {
+    "name": "GROUP_EGG_SNAKE_M",
+    "type": "monstergroup",
+    "monsters": [ { "monster": "mon_rattlesnake_big_s" } ]
```

Comment:
I wanna say you can remove those entirely if you got rid of eggs spawning rattlesnakes. If you wanna be brave and get rid of the first one check in all of JSON if it's not used anywhere, though, please.

Reply:
Yeah, I think I'll axe that. Those groups are not used anywhere anyway.